envs/monkey_zoo/blackbox/utils/gcp_machine_handlers.py

```diff
@@ -1,35 +1,47 @@
 import logging
+import os
 import subprocess
 
 LOGGER = logging.getLogger(__name__)
 
 
 class GCPHandler(object):
-
+    # gcloud commands
```

Comment:
Since each of those commands starts with "gcloud", I'm not sure what value this comment adds.